netutils/mac.py

```diff
@@ -136,3 +136,29 @@ def mac_normalize(mac: str) -> str:
         if char in mac:
             mac = mac.replace(char, "")
     return mac
+
+
+@_valid_mac
+def get_oui(mac: str) -> str:
+    """Returns the company name for a given mac as defined by the IEEE.
+
+    Args:
+        mac: A MAC address in string format that matches one of the defined regex patterns.
+
+    Returns:
+        The name of the company the mac is related to.
+
+    Example:
+        >>> from netutils.mac import get_oui
+        >>> from netutils.constants import OUI_MAPPINGS
+        >>> get_oui("cc.79.d7.dd.ee.ff")
+        'Cisco Systems, Inc'
+        >>>
+    """
+    normalized_mac_prefix = mac_normalize(mac)[0:6]
```

Comment:
I know the pattern created already uses constants.py, that being said... perhaps we should consider running the load of the json file here and doing it intelligently? 
```
OUI_MAPPINGS = {}
@_valid_mac
def get_oui(mac: str) -> str:
    if not OUI_MAPPINGS:
        with open("/".join([dirname(netutils_file), "oui_mappings.json"]), encoding="utf-8") as fh:
            OUI_MAPPINGS: typing.Dict[str, str] = json.loads(fh.read())
    oui_company = OUI_MAPPINGS.get(normalized_mac_prefix)

    normalized_mac_prefix = mac_normalize(mac)[0:6]
    if not oui_company:
        raise ValueError(f"There was no matching entry in OUI_MAPPINGS for {normalized_mac_prefix}")

    return oui_company
```
With this pattern, you only load into memory if you actually use the function.